lrathoni/Lullaby-for-a-fish
# Dev cam arm

**What :** 
Hand and flute interaction added to the camera. If you play a note, the flute will appear in front of the player.
If you don't play a note in a certain amount of time, the flute will disappear. 

**How to test :**
Press Y, U, I, O, P keys
See if the flute is changing positions depending of the key pressed
Wait a moment ( 3 secondes ) to see if the flute disappears
